src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -178,7 +179,7 @@ bool MulticopterLandDetector::_get_landed_state()
 
 	// Return status based on armed state and throttle if no position lock is available.
 	if (_vehicleLocalPosition.timestamp == 0 ||
-	    hrt_elapsed_time(&_vehicleLocalPosition.timestamp) > 500000 ||
+	    hrt_elapsed_time(&_vehicleLocalPosition.timestamp) > 0.5e6 ||
```

Comment:
This leads to a double executed in soft-float (longer runtime, more stack). Should be reverted.